docs/clients/dovecot.txt

```diff
@@ -45,6 +45,26 @@ Configuring Dovecot to work with Virtual Exim
    Dovecot expects the select to return a set of columns with particular
    names, hence usage of the 'AS' keyword above.
 
-3. There's no step 3! Just restart Dovecot, and it should work.
+3. Exim can’t directly handle SHA512-crypt for user authentication but exim can use
```

Comment:
Same here. If most current Linux distros support SHA-512 out of the box, we could use that as an excuse not to tell Exim to authenticate against other products by default